src/cmd-generate-release-meta

```diff
@@ -218,7 +218,8 @@ def append_build(out, input_):
     if i is not None:
         ext = get_extension(i['path'], 'qemu-secex', arch)
         arch_dict["media"]["qemu"]["artifacts"][f"secex.{ext}"] = {
-            "disk": artifact(i)
+            "disk": artifact(i),
+            "ignition-gpg-key": url_builder(out.get('stream'), out.get('release'), arch, "ignition.gpg.pub")
```

Comment:
Hmm, I'm not sure if this belongs under `secex.{ext}`. The key is independent of the compression format used for the qemu image. Let's put it down as another artifact instead? (I.e. at one level up.)

Also, I'm thinking we should include the sha256 and on FCOS also sign the pubkey with the Fedora key. This is more easily done if the key is promoted to another entry in the `images` dict of `meta.json`.

So then the `.architectures.s390x.media["qemu-secex"]` key in the release meta would look something like:

```json
{
  "artifacts": {
    "qcow2.xz": {
      "disk": {
        "location": "...",
        "signature": "...",
        "sha256": "...",
        "uncompressed-sha256": "..."
      }
    },
    "ignition-gpg-pubkey": {
      "location": "...",
      "signature": "...",
      "sha256": "...",
      "uncompressed-sha256": null
    }
  }
}
```

/cc @bgilbert Thoughts?

Reply:
if so, than i'd need to update `schema` , should i?